docs/source/usage/security_recommendations.rst

```diff
@@ -112,5 +112,177 @@ Note:
+
+.. prompt:: bash $
+
+  openstack baremetal node set \
+    --driver idrac \
+    --driver-info drac_username=user \
+    --driver-info drac_password=pa$$w0rd
+    --driver-info drac_address=drac.host \
+    --driver-info redfish_username=user \
+    --driver-info redfish_password=pa$$w0rd \
+    --driver-info redfish_address=drac.host \
+    --driver-info redfish_system_id=/redfish/v1/Systems/System.Embedded.1 \
+    --bios-interface idrac-redfish \
+    --inspect-interface idrac-redfish \
+    --management-interface idrac-redfish \
+    --power-interface idrac-redfish
+    <node>
+
+
+Import or export iDRAC configuration with idrac-redfish:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Sample scripts to export and import the iDRAC configuration using the `Redfish API`_ are shown below. For more information refer the Redfish Module `function definitions`_.
+
+* Export configuration:
+
+.. prompt:: bash $
+
+  python3 ExportSystemConfigurationLocalREDFISH.py -ip '10.1.10.3' -u 'new-user' -p $ADMIN_PASSWORD -t IDRAC
+
+
+* Import configuration after making changes:
+
+.. prompt:: bash $
+
+  python3 ImportSystemConfigurationLocalREDFISH.py -ip '10.1.10.3' -u 'new-user' -p $ADMIN_PASSWORD -t IDRAC
+
+
+Note:
+ * Redfish does not support exporting the iDRAC license iDRAC 7/8, RACADM command can be used instead.
+      * Usage
+           - racadm license help import
+           - racadm license help export
+
+Import or export iDRAC configuration with Ironic manual cleaning :
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The clean and deploy steps provided in this section allow an owner to configure the system and collect the system inventory using configuration mold files. To export or import the configuration of current system or replicate the system configuration to that of a similar system.
+
+The following is an example of a system configuration JSON mold:
+
+.. prompt:: bash $
+
+  {
+  "oem": {
+    "interface": "idrac-redfish",
+    "data": {
+      "SystemConfiguration": {
+        "Model": "PowerEdge R640",
+        "ServiceTag": "8CY9Z99",
+        "TimeStamp": "Fri Jun 26 08:43:15 2020",
+        "Components": [
+          {
+            [...]
+            "FQDD": "NIC.Slot.1-1-1",
+            "Attributes": [
+              {
+              "Name": "VirtMacAddr",
+              "Value": "00:00:00:00:00:00",
+              "Set On Import": "False",
+              "Comment": "Read and Write"
+              },
+              {
+              "Name": "VirtualizationMode",
+              "Value": "NONE",
+              "Set On Import": "True",
+              "Comment": "Read and Write"
+              },
+            [...]
+            ]
+          }
+        ]
+      }
+    }
+  }
+
+
+**Steps**:
+
+1. Storage options for Configuration settings:
+
+   * Swift Containers (Admins):
```

Comment:
I'd say "Admin-only" and then add an explanation as to why.

Reply:
Done